CMakeLists.txt

```diff
@@ -150,6 +150,11 @@ IF (MSVC AND CMAKE_GENERATOR MATCHES "NMake")
   SET (USING_NMAKE TRUE)
 ENDIF (MSVC AND CMAKE_GENERATOR MATCHES "NMake")
 
+IF (MSVC AND CMAKE_GENERATOR MATCHES "Ninja")
```

Comment:
Thanks.

On Mon, May 23, 2016 at 8:27 PM, Matthias Kuhn <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/qgis/QGIS/pull/3092#discussion_r64200173>:
>
> > @@ -150,6 +150,11 @@ IF (MSVC AND CMAKE_GENERATOR MATCHES "NMake")
> >    SET (USING_NMAKE TRUE)
> >  ENDIF (MSVC AND CMAKE_GENERATOR MATCHES "NMake")
> >
> > +IF (MSVC AND CMAKE_GENERATOR MATCHES "Ninja")
>
> IF (CMAKE_GENERATOR MATCHES "Ninja")
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/qgis/QGIS/pull/3092/files/efef9b2f27b1b726f2d87942d7b964ae2e870f0a#r64200173>
>